docs/API.md

```diff
@@ -45,14 +45,19 @@ In any case a user doesn't have access to a requested entity, a 403 error will b
 ### If-Modified-Since
 
 Some index endpoints support limiting the result set to entries that have been changed since the given time.
+The supported date formats are:
+
+* IMF-fixdate:      `Sun, 03 Aug 2019 10:34:12 GMT`
```

Comment:
Could you try to highlight tat IMF-fixdate should be used (the others are considered obsolete) according to https://tools.ietf.org/html/rfc7231#section-7.1.1.1

Reply:
Done! See [this commit](https://github.com/nextcloud/deck/pull/1196/commits/b7a36851de82542f7634e68356a22bc96cdf4f20).